app/models/member.rb

```diff
@@ -22,9 +22,11 @@ def name=(full_name)
   end
 
   def liquid_data
+    full_name = name
     attributes.merge({
-      name: name,
-      welcome_name: name.blank? ? email : name
+      name: full_name,
```

Comment:
Don't quite understand... our current solution for member names is that campaigners can ask for it however they want, we store it in two columns in the database, and we serialize it both ways. The redundancy of `name` and `full_name` is for if someone calls their field `full_name` instead of the AK matching `name` (as paul did when testing)